tools/src/dbgen/Lib.hs

```diff
@@ -30,52 +35,108 @@ import           Pos.Wallet.Web.ClientTypes
 import           Pos.Wallet.Web.ClientTypes.Instances ()
 import           Pos.Wallet.Web.Methods.Logic
 import           Pos.Wallet.Web.Methods.Restore
+import           Pos.Wallet.Web.State.State           (getWalletUtxo, setWalletUtxo, updateWalletBalancesAndUtxo)
+import           Test.QuickCheck                      (arbitrary, generate)
 import           Rendering
 import           Text.Printf
 
 --
 -- Types
 --
 
-data GenSpec = GenSpec {
-    wallets       :: !Integer
+-- | A simple example of how the configuration looks like.
+_testSpec :: GenSpec
+_testSpec = GenSpec
+    { walletSpec = WalletSpec
+        { accounts = 1
+        , accountSpec = AccountSpec { addresses = 100 }
+        , fakeUtxo = FakeUtxoSpec { amount = 1000, distribution = RangeDistribution { range=100 } }
+        }
+    , wallets = 1
+    }
```

Comment:
Good idea, I like this! I would call it simply `_exampleSpec` as we are not really testing anything, but that's a minor nitpick.